plugins/Coinflip/plugin.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "Coinflip",
+  "version": "0.0.0",
```

```suggestion
  "version": "0.1.0",
```
starting from `0.1.0` is usually common